crates/ra_hir/src/resolve.rs

```diff
@@ -128,6 +132,30 @@ impl Resolver {
         resolution
     }
 
+    pub fn resolve_macro_call(
+        &self,
+        db: &impl HirDatabase,
+        name: &Name,
```

Comment:
Let's pass an `hir::Path` here? Currently we handle only single-segment macro path, but hopefully that'll change!